docs/source/usrman/envvars.rst

```diff
@@ -0,0 +1,60 @@
+.. _environment:
+
+Environment variables
+=====================
+
+Below we list the environment variables added since mpi4py v3.1.0. The environment
+variables overwrite their corresponding Python attributes at the ``MPI`` import time.
```

Comment:
"... at MPI import time", that is, remove "the".  Or perhaps rephrase as "at import time of the MPI module". I believe the second is a bit more explicit. Sorry I did not catch this one before.

Reply:
Done.